keras/preprocessing/image.py

```diff
@@ -298,7 +308,7 @@ def load_img(path, grayscale=False, target_size=None):
         img = img.convert('L')
     else:  # Ensure 3 channel even when loaded image is grayscale
         img = img.convert('RGB')
-    if target_size:
+    if target_size[0] and target_size[1]:  # Can't pass None in from iterator, hence tuple
```

Comment:
We can keep the default to `None` and convert to `(None, None)` here. At the very least, `target_size=None` should keep being supported (for backwards compatibility).